src/utils.js

```diff
@@ -138,16 +138,31 @@ export function setCSSVariables (transition, currentStage, instance) {
   })
 }
 
-export async function renderView (el, canvas, views, init) {
+export async function renderView (el, canvas, views, init, componentContext) {
     var viewName = null
     init ? viewName = el : viewName = el.dataset.to
     var newView = document.createElement('template')
-    // makes optional de 'render' function
-    typeof views[viewName] === 'object' && views[viewName].render !== undefined
-      ? newView.innerHTML = await views[viewName].render()
-      : newView.innerHTML = views[viewName]
+    
+    if(typeof views[viewName] === 'object' && views[viewName].render !== undefined) {      
+      // makes optional de 'render' function
+      newView.innerHTML = await views[viewName].render()
+    } else if(typeof views[viewName] === 'function') {
+      // view is a component constructor
+      var newViewInner = document.createElement('div')
```

Comment:
I'm adding this wrapping div since I ran into problems having Svelte render into a <template> and I also didn't want to change `newView` to a div to not break other use-cases.

But because of this extra div, I also had to wrap the `this.canvas.removeChild(element)` line in zumly.js below and also try removing the parent...